pl_bolts/models/self_supervised/amdim/networks.py

```diff
@@ -144,8 +144,16 @@ def forward(self, x):
 
 
 class Conv3x3(nn.Module):
-    def __init__(self, n_in, n_out, n_kern, n_stride, n_pad,
-                 use_bn=True, pad_mode='constant'):
+    def __init__(
+            self,
+            n_in: int,
+            n_out: int,
+            n_kern: int,
+            n_stride: int,
+            n_pad: int,
+            use_bn: bool = True,
+            pad_mode: srt = 'constant',
```

Comment:
What is this srt type? getting an error on it not being imported/available

Reply:
@djbyrne should be str

Reply:
I have fixed it and pushed fix but canot see it here...